### PR TITLE
oxker 0.7.0 (new formula)

### PR DESCRIPTION
**Pull request by miketheman**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/o/oxker.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    # oxker is a TUI application, and has no CLI functionality
+    assert_match version.to_s, shell_output("#{bin}/oxker --version")
```
Comment (p-linnane) [inline Formula/o/oxker.rb:16]:
```suggestion
    assert_match version.to_s, shell_output(bin/"oxker --version")

    assert_match "a value is required for '--host <HOST>' but none was supplied",
      shell_output(bin/"oxker --host 2>&1", 2)
```

File: Formula/o/oxker.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class Oxker < Formula
+  desc "Terminal User Interface (TUI) to view & control docker containers"
+  homepage "https://github.com/mrjackwills/oxker"
+  url "https://github.com/mrjackwills/oxker/archive/refs/tags/v0.7.0.tar.gz"
+  sha256 "082c152e31fb0c0f0adfa60780480d756c4589649963b6616599ab696b7989b4"
+  license "MIT"
+
```
Comment (p-linnane) [inline Formula/o/oxker.rb:7]:
```suggestion
head "https://github.com/mrjackwills/oxker.git", branch: "main"

```

File: Formula/o/oxker.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+class Oxker < Formula
+  desc "Terminal User Interface (TUI) to view & control docker containers"
+  homepage "https://github.com/mrjackwills/oxker"
+  url "https://github.com/mrjackwills/oxker/archive/refs/tags/v0.7.0.tar.gz"
+  sha256 "082c152e31fb0c0f0adfa60780480d756c4589649963b6616599ab696b7989b4"
+  license "MIT"
+head "https://github.com/mrjackwills/oxker.git", branch: "main"
```
Comment (miketheman) [inline Formula/o/oxker.rb:7]:
```suggestion
  head "https://github.com/mrjackwills/oxker.git", branch: "main"
```

**Comment by p-linnane** [comment]
Thanks @miketheman. Please squash the commits with the message `oxker 0.7.0 (new formula)`.

**Review by chenrui333 — APPROVED** [review]

**Review by p-linnane — APPROVED** [review]

**Pull request merged**